evm/deploy-its.js

```diff
@@ -337,6 +346,10 @@ async function deployAll(config, wallet, chain, options) {
             continue;
         }
 
+        if (isHyperliquidChain(chain) && shouldUseBigBlocks(key)) {
```

Comment:
hmm, i don't like this separate configuration for when to use big blocks. Add an optional `useHyperliquidBigBlocks` field in the contract specific config above so you can just test if that is defined?